### PR TITLE
Make luigi package more flake8 compliant

### PR DESCRIPTION
**Pull request by toidi**
Motivation of this PR was to make flake8 checks as strict as possible.

Before: 
```
commands = flake8 --max-line-length=384 --exclude=doc,test,luigi/six.py
  flake8 --max-line-length=100 --ignore=E265 doc
  flake8 --max-line-length=252 --ignore=F401,F841 test
```
After:

```
commands = flake8 --max-line-length=160 --exclude=doc,luigi/six.py
   flake8 --max-line-length=100 --ignore=E265 doc
```

This will protect luigi package from govnokod.

In future `max-line-length` should be less then 120 chars for better readability.

### CODE REVIEW

File: luigi/execution_summary.py [hunk 1]
```diff
@@ -97,7 +101,8 @@ def _get_str(task_dict, extra_indent):
             break
         if len(tasks[0].get_params()) == 0:
             row += '- {0} {1}()'.format(len(tasks), str(task_family))
-        elif _get_len_of_params(tasks[0]) > 60 or (len(tasks) == 2 and len(tasks[0].get_params()) > 1 and (_get_len_of_params(tasks[0]) > 40 or len(str(tasks[0])) > 100)) or len(str(tasks[0])) > 200:
+        elif _get_len_of_params(tasks[0]) > 60 or len(str(tasks[0])) > 200 or \
```
Comment (Tarrasch) [inline luigi/execution_summary.py]: I thought `\` were unpythonic and one preferred parenthesis? No?

**Comment by Tarrasch** [comment]
Awesome progress, for future reference, this a continuation of #1174.

**Comment by Tarrasch** [comment]
Thanks so much for caring about the code quality of luigi! :)

**Comment by Tarrasch** [comment]
What's "govnokod" by the way? I only found results in Russian. :)

File: luigi/execution_summary.py (hunk 1 above)
Comment (toidi) [inline luigi/execution_summary.py]: I guess there is no more or less pythonic. In my opinion - pythonic mean better readable.

In some cases parenthesis better readable but in this case - this condition `\` looks good. Condition must no be over-parenthesised.


https://www.python.org/dev/peps/pep-0008/#maximum-line-length
> Backslashes may still be appropriate at times. For example, long, multiple with -statements cannot use implicit continuation, so backslashes are acceptable:

**Comment by toidi** [comment]
> Thanks so much for caring about the code quality of luigi! :)
We use `luigi` and loved it. I wish bright future for project. You are welcome.

> What's "govnokod" by the way?
Well.. It is very specific term. Usually govnokod generated by anything but not developers head. It is like monkey code but a little bit different.

Talk is cheap. I show you the code: http://govnokod.ru :-))

File: test/execution_summary_test.py [hunk 2]
```diff
@@ -166,8 +173,17 @@ def run(self):
...
+                      '* 2 were left pending, among these:\n'
+                      '    * 1 were being run by another worker:\n'
+                      '        - 1 LockTask()\n'
+                      '    * 1 had dependencies that were being run by other worker:\n'
+                      '        - 1 ParentTask()\n', s)
+        self.assertIn('\n\nThe other workers were:\n'
+                      '    - other_worker ran 1 tasks\n\n'
+                      'Did not run any tasks\n'
+                      'This progress looks :) because there were no failed '
+                      'tasks or missing external dependencies\n', s)
```
Comment (Tarrasch) [inline test/execution_summary_test.py]: Ah this looks so much nicer! :)

**Comment by Tarrasch** [comment]
This is *so* awesome! Thanks :)

**Pull request merged**